src/promptflow-evals/promptflow/evals/evaluators/f1_score/__init__.py

```diff
@@ -9,30 +9,32 @@
 from pathlib import Path
 
 
-def init():
+class F1ScoreEvaluator:
     """
-    Initialize an evaluation function for calculating F1 score.
+        Initialize an evaluation function for calculating F1 score.
 
-    :return: A function that evaluates F1 score.
-    :rtype: function
+        :return: A function that evaluates F1 score.
+        :rtype: function
 
-    **Usage**
+        **Usage**
 
-    .. code-block:: python
+        .. code-block:: python
 
-        eval_fn = f1_score.init()
-        result = eval_fn(
-            answer="The capital of Japan is Tokyo.", 
-            ground_truth="Tokyo is Japan's capital, known for its blend of traditional culture \
-                and technological advancements.")
-    """
-    def eval_fn(answer: str, ground_truth: str):    
```

Comment:
Looks like if we move the `load_flow` outside of `eval_fn`, and then only calls `f(answer=answer, ground_truth=ground_th)` inside `eval_fun`, the effect will be same as doing it as a class - load_flow will be only called once for batch run.